r/R/duckdb.R

```diff
@@ -55,7 +55,7 @@
 to_duckdb <- function(.data,
                       con = arrow_duck_connection(),
                       table_name = unique_arrow_tablename(),
-                      auto_disconnect = FALSE) {
+                      auto_disconnect = TRUE) {
   .data <- as_adq(.data)
   duckdb::duckdb_register_arrow(con, table_name, .data)
 
```

Comment:
Comment on line 70:
> # we should probably confirm that this use of src$disco is kosher.

Is this now confirmed?

Reply:
There hasn't been any comments (positive or negative) on https://github.com/tidyverse/dbplyr/issues/681 

On the positive side: this is "just" an element of a lis that stores an object and we are relying on the way R cleans up objects when they are GCed, we could name it anything and so long as d(b)plyr doesn't collide with that name for something else it should _just work_ whether or not we are supposed to extend `tbl$src$disco`

Reply:
_nods_ yes, totally could. I kept it in the "traditional" spot since that's what dbplyr does, but maybe it would be better to proactively avoid any possible collisions/changes in their API for that.

NGL, `disco` is a hilariously awesome name even if it isn't _super_ transparent 🕺